frontend/src/components/chat/chat-panel.tsx

```diff
@@ -296,6 +295,18 @@ const ChatPanelBody = () => {
 
   const isLoading = status === "submitted" || status === "streaming";
 
+  // Scroll to the latest chat message at the bottom
+  useEffect(() => {
+    const scrollToBottom = () => {
+      if (scrollContainerRef.current) {
+        const container = scrollContainerRef.current;
+        container.scrollTop = container.scrollHeight;
+      }
+    };
+
+    requestAnimationFrame(scrollToBottom);
+  }, [messages, isLoading]);
```

Comment:
I used to have this but folks didn't like it so I removed it. I think we can maybe just run it when the activeChatId changes